### PR TITLE
Fix update failures due to package with interactive prompt

### PR DESCRIPTION
**Pull request by jcapona**
| Status  | Ticket/Issue |
| :---: | :--: |
| Ready | [Ticket](https://pi-top.atlassian.net/browse/OS-1469) |

#### Main changes
-

#### Screenshots (feature, test output, profiling, dev tools etc)

[insert screenshots here]

#### Other notes (e.g. implementation quirks, edge cases, questions / issues)
-

#### Manual testing tips
-

#### Tag anyone who definitely needs to review or help
-

### CODE REVIEW

File: pt_os_web_portal/pt-os-web-portal [hunk 1]
```diff
@@ -7,7 +7,7 @@ import pitop.common.ptdm  # noqa: E402
 import zmq.green  # noqa: E402
 
 # Use zmq.green for gevent compatibility
-pitop.common.ptdm.zmq = zmq.green
+pitop.common.ptdm.import_zmq = lambda: zmq.green
```
Comment (angusjfw) [inline pt_os_web_portal/pt-os-web-portal:10]: could be worth keeping both forms of this in case we change it again and directly import zmq

**Review by angusjfw — APPROVED** [review]
looks good, needs thorough testing on bullseye and buster, new OS, older OS, second update etc
